arangosh/Restore/RestoreFeature.cpp

```diff
@@ -1164,7 +1164,9 @@ arangodb::Result processInputDirectory(
         if (result.fail()) {
           return result;
         }
+      }
 
+      if (progressTracker.getStatus(name.copyString()).state < arangodb::RestoreFeature::CREATED) {
```

Comment:
we could change `getStatus` to expect a `std::string_view` but I guess this is fine.

Reply:
Yes, we should do this, but seperately.